main.go

```diff
@@ -264,6 +264,26 @@ The value 'none' can be used to disable access logs.`),
 			Usage: "Dial backend request timeout in seconds",
 		},
 		cli.IntFlag{
+			Name:  "client-read-timeout",
+			Value: 30, // TODO: TBC
```

Comment:
I'd prefer to have the default value for these new flag set to `0`. This way we don't alter the default behavior for planb when starting a new instance without setting them.